workspaces/cli-config/src/helpers/ignore-parser.ts

```diff
@@ -1,17 +1,17 @@
-import * as pathToRegexp from 'path-to-regexp';
+import { pathToRegexp } from 'path-to-regexp';
 
 export interface IIgnoreRunnablePredicate {
-  methods: string[],
-  path: string,
-  regex: RegExp,
+  methods: string[];
+  path: string;
+  regex: RegExp;
 
-  shouldIgnore(method: string, url: string): boolean
+  shouldIgnore(method: string, url: string): boolean;
 }
 
 export interface IIgnoreRunnable {
-  rules: IIgnoreRunnablePredicate[],
+  rules: IIgnoreRunnablePredicate[];
 
-  shouldIgnore(method: string, url: string): boolean
+  shouldIgnore(method: string, url: string): boolean;
 }
 
```

Comment:
Is the Intent here that ignore rules run on the agent-cli before sending?

Reply:
There is a filter that the proxy uses to discard interactions that it knows should not be logged. All other interactions are recorded, and then this filter is used to limit which interactions are considered for diffs. This lets you adjust the filters without needing to re-do a capture